### PR TITLE
Insights2

### PR DESCRIPTION
**Pull request by vinkobedek**

### CODE REVIEW

File: how-to/spdockit-insights/search-alerts.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Search Alerts
+description: How to use alerts
+author: Vinko Bedek
+date: 22/5/2017
+---
+<!--
+1. treba napomenuti za licence, ali nisam siguran kak tocno 
+-->
+SPDocKit Insights provides the ability to be notified when a search query is satisfied. This is done as the events are being collected from the servers. When defining alerts, you can target a single farm, or apply the alert to all of the farms contained within the SPDocKit database. Once sent, the alert for a farm is not evaluated again for a configurable back off period. Without the back off period, our mailboxes would suffer greatly once a problem arises.
```
Comment (tomislavkunaj) [inline how-to/spdockit-insights/search-alerts.md]: our mailboxes? I'd remove _our_ completely.

Comment (tomislavkunaj) [inline how-to/spdockit-insights/search-alerts.md]: _or apply the alert to the farms contained_ - _of the_ seems superfluous

Comment (tomislavkunaj) [inline how-to/spdockit-insights/search-alerts.md]: _is not evaluated again for a configurable back off period._ - also sounds clunky. Maybe something along the lines:
_Once sent, the alert for a farm will not be evaluated again within a certain back off time period, which you can configure._
Not sure that doesn't sound clunky as well.

File: how-to/spdockit-insights/use-spdockit-insights.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+## Use SPDocKit to view farm performance
+
+SPDocKit Insights collects performance data[*](#licenseLimitations) by default. This behavior can be modified in the [settings](#internal/how-to/spdockit-insights/customize-settings/).  
+To view the collected data, select the performance tab in the left navigation.  
+The default dashboard has three views:
+1. Compact
+2. Tiles
+3. Grid 
+
+Most performance counters collected have a warning and critical limit based on the values collected within a certain time period. This will impact the status of the server visible on the dashboards. Since most counters are not visible on the main dashboard, additional dashboards with more details are available:
```
Comment (tomislavkunaj) [inline how-to/spdockit-insights/use-spdockit-insights.md]: Not sure what _visible_ refers to here?
_This will impact the displayed status of the server on the dashboards._ maybe better?

File: how-to/spdockit-insights/search-alerts.md (hunk 1 above)
Comment (vinkobedek) [inline how-to/spdockit-insights/search-alerts.md]: changed it.

File: how-to/spdockit-insights/use-spdockit-insights.md (hunk 2 above)
Comment (vinkobedek) [inline how-to/spdockit-insights/use-spdockit-insights.md]: visible refers to the fact that you only see the four main counters on the default dashboard

Comment (vinkobedek) [inline how-to/spdockit-insights/use-spdockit-insights.md]: sorry, wrong visible commented.

Comment (vinkobedek) [inline how-to/spdockit-insights/use-spdockit-insights.md]: changed it.

**Pull request merged**